client/src/js/modules/shipment/views/shipment.js

```diff
@@ -279,10 +308,22 @@ define(['marionette',
                     this.$el.find(".remoteform").show()
                 }
             }
+            if (dynamic === 'Ready' || (app.proposal && app.proposal.get('ACTIVE') != '1')) {
```

Comment:
Is it worth keeping the proposal active comparison as is? With how JS does these comparisons, 1 == 1, but 1 == "1", "1" == true, and so on